tests/frontend/casperjs.conf.js

```diff
@@ -26,7 +27,7 @@ module.exports = {
     phantomcss: function () {
         phantomcss.init({
             libraryRoot: './node_modules/phantomcss',
-            screenshotRoot: './tests/screenshots/reference',
+            screenshotRoot: system.env.SCREENSHOT_REFERENCES,
```

Comment:
would `process.env.SCREENSHOT_REFERENCES` not work?

Reply:
nope sadly not...